test/01_test_text_utils.jl

```diff
@@ -172,47 +175,60 @@ id est laborum."""
-            "┌────────────────┬───────────────\n─┬────────────────┬─────────────\n───┬──────────────",
+            "┌────────────────┬───────────────\n─┬────────────────┬──────────────\n──┬──────────────",
         ),
         (
             "┌────────────abcde────┬──────────── ────┬────────abcde fghi────────┬────────────────┬──────────────",
-            "┌────────────abcde────┬──────────\n── ────┬────────abcde \nfghi────────┬────────────────┬──\n────────────",
+            "┌────────────abcde────┬──────────\n── ────┬────────abcde \nfghi────────┬────────────────┬───\n───────────",
         ),
         (
             "┌─────────{red}───ab{/red}cde────┬──────{green}────── ────┬────────abcde fghi{/green}────────┬────────────────┬──────────────",
-            "┌─────────\e[31m───ab\e[39mcde────\n┬──────\e[32m────── \n────┬────────abcde \nfghi\e[39m────────┬───────────\n─────┬──────────────",
+            "┌─────────\e[31m───ab\e[39mcde────\n┬──────\e[32m────── \n────┬────────abcde \nfghi\e[39m────────┬────────────\n────┬──────────────",
         ),
         (
             "┌──────────{red}────{/red}──┬{blue bold}────────────────┬──{/blue bold}──────────────┬────────────────┬──────────────end",
-            "┌──────────\e[31m────\e[39m──┬\e[1m───────────────┬──{/blue\n}\e[22m\e[39m}──────────────┬────────\n────────┬──────────────end\e[39m",
+            "┌──────────\e[31m────\e[39m──┬\e[1m───────────────┬──{/blue}\n\e[22m\e[39m}──────────────┬──────────\n──────┬──────────────end\e[39m",
         ),
         (
             "."^100,
-            ".................................\n................................\n................................\n...",
+            ".................................\n.................................\n.................................\n.",
         ),
         (
             ".{red}|||{/red}...."^10,
-            ".\e[31m|||\e[39m.....\e[31m|||{/red\n}.....\e[31m|||\e[39m.....\e[31m||\n|\e[39m\e[31m.....\e[31m|||\e[39m.....\e[31m|||\e[39m\e[31m.....\e[31m|||\e[39m\e[31m\n.....\e[31m|||\e[39m\e[31m.....\e[31m|||\n\e[39m\e[31m.....\e[31m|||\e[39m\e[31m....\e[39m",
+            ".\e[31m|||\e[39m.....\e[31m|||{/red\n}.....\e[31m|||\e[39m.....\e[31m|||\n\e[39m\e[31m.....\e[31m|||\e[39m.....|||\e[39m\e[31m.....\e[31m|||\e[39m\e[31m...\n..\e[31m|||\e[39m\e[31m.....\e[31m|||\e[39m.....\e[31m|||\e[39m\e[31m....\e[39m",
         ),
         (
             ".|||...."^10,
-            ".|||.....|||.....|||.....|||.....\n|||.....|||.....|||.....|||.....\n|||.....|||....",
+            ".|||.....|||.....|||.....|||.....\n|||.....|||.....|||.....|||.....|\n||.....|||....",
         ),
         (str, str_reshaped),
         (logo_str, logo_str_reshaped),
     ]
 
-    for (s1, s2) in strings
-        reshaped = reshape_text(s1, 33)
-        @test reshaped == s2
-    end
-
-    for width in (40, 60, 99)
-        rh = reshape_text(str, width)
-        @test !any(textlen.(split(rh, "\n")) .> width)
+    width = 33
```

Comment:
Can you walk me through what's happening in these lines?

Reply:
Ha sorry, they were moved above the `enumerate(strings)` test. I'm going to restore the testing order ...